docs/ci/codemagic.mdx

```diff
@@ -0,0 +1,231 @@
+---
+sidebar_position: 2
+title: ♦️ Codemagic
+description: Integrate Shorebird into your Codemagic workflow
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Codemagic Workflow Integration
+
+This guide will help you integrate Shorebird into your Codemagic Workflow using the Codemagic Workflow Editor and YAML configuration.
+
+## Prerequisites
+
+✅ Shorebird CLI is installed on your machine
+
+✅ You are logged into a paid account.
```

Comment:
I don't think still true. Just copied it from GitHub docs.

Reply:
Yeah that's no longer the case so I think we can reword to "You are logged into a shorebird account"